Sources/ApolloCodegenLib/Frontend/GraphQLSchema.swift

```diff
@@ -63,18 +64,18 @@ public class GraphQLEnumValue: JavaScriptObject {
 
 public class GraphQLInputObjectType: GraphQLNamedType {
   private(set) lazy var description: String? = self["description"]
-  
-  private(set) lazy var fields: [String: GraphQLInputField] = try! invokeMethod("getFields")
+
+  lazy var fields: OrderedDictionary<String, GraphQLInputField> = try! invokeMethod("getFields")
 }
 
 public class GraphQLInputField: JavaScriptObject {
-  private(set) lazy var name: String = self["name"]
+  lazy var name: String = self["name"]
   
-  private(set) lazy var type: GraphQLType = self["type"]
+  lazy var type: GraphQLType = self["type"]
   
   private(set) lazy var description: String? = self["description"]
   
-  private(set) lazy var defaultValue: Any? = self["defaultValue"]
+  lazy var defaultValue: Any? = self["defaultValue"]
```

Comment:
Why are these all no longer `private (set)`?

Reply:
If the setter is private then the mocking functions can't work.